canonical-docs/ubuntu-core-docs
# Entrypoint for talisker gevent workers

Fixes https://github.com/ubuntudesign/base-squad/issues/99

QA
--

``` bash
./run
```

See that it mentions "Booting worker" 3 times. Go to http://localhost:8207 and check everything works just as well as it used to (many broken links in navigation for some reason). Now:

``` bash
docker build --tag core-docs --build-arg COMMIT_ID=`git rev-parse HEAD` .
docker run -ti -p 8099:80 core-docs
```

Go to http://localhost:8099 and similarly see that it mentions "Booting worker" 3 times, and check everything still works just as well as it used to.

Also, check the `X-VCS-Revision` shows correctly:

``` bash
$ curl -Is localhost:8099 | grep VCS
X-VCS-Revision: {commit_hash}
```
